solutions/src/main/java/ru/sberbank/school/task09/RouteServiceImpl.java

```diff
@@ -11,18 +13,44 @@
  * Created by Mart
  * 18.06.2019
  **/
+@Solution(9)
 public class RouteServiceImpl extends RouteService<City, Route<City>> {
-    private File caheFile;
-    private RouteSerializableManager serializableManager;
+    private RouteSerializeManager serializeManager;
 
-    public RouteServiceImpl (@NonNull String filepath, RouteSerializableManager serializableManager) {
+    public RouteServiceImpl (@NonNull String filepath, RouteSerializeManager serializeManager) {
```

Comment:
&#39;(&#39; is preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1856642286>*